includes/admin/class-admin.php

```diff
@@ -1162,6 +1173,33 @@ private function check_deprecation_warnings() {
             }
         }
 
+		/**
+		 * Check old inline fontawesome setting.
+		 * If it was enabled by default, we show an admin notice.
+		 * On dismiss we keep the setting.
+		 *
+		 * @since x.x
+		 */
+		private function check_font_awesome_setting() {
+			if ( ! current_user_can( 'administrator' ) ) {
```

Comment:
Let's use install_plugins here.

We could also include `! WPBDP_App_Helper::is_bd_page()` in the same line too.